Modules/_sqlite/connection.c

```diff
@@ -569,6 +567,10 @@ _pysqlite_set_result(sqlite3_context* context, PyObject* py_val)
         sqlite3_result_blob(context, view.buf, (int)view.len, SQLITE_TRANSIENT);
         PyBuffer_Release(&view);
     } else {
+        callback_context *ctx = (callback_context *)sqlite3_user_data(context);
+        PyErr_Format(ctx->state->ProgrammingError,
+                     "UDF's cannot return '%s' values to SQLite",
```

Comment:
```suggestion
                     "User-defined functions cannot return '%s' values to SQLite",
```
No other error message uses the "UDF" abbreviation.

Reply:
Yes, this is a good opportunity to get rid of those abbreviations!